cannon/mipsevm/exec/mips_instructions.go

```diff
@@ -338,10 +338,10 @@ func ExecuteMipsInstruction(insn uint32, opcode uint32, fun uint32, rs, rt, mem
 		switch opcode {
 		// SPECIAL2
 		case 0x1C:
-			switch {
-			case fun == 0x2: // mul
+			switch fun {
+			case 0x2: // mul
```

Comment:
Are all of these format changes really required? cannon seems to compile fine for me even with go.mod modified to use go 1.24.

I believe a lot of the formatting choices in cannon were designed so that it was as similar as possible to the solidity version (which has to have exactly identical behaviour).